pkg/core/migration/migration.go

```diff
@@ -99,11 +99,20 @@ const DefaultSystemLabel string = "Managed by CM-Beetle"
 func CreateVMInfra(nsId string, infraModel *mci.TbMciDynamicReq) (mci.TbMciInfo, error) {
 
 	client := resty.New()
-	client.SetBasicAuth("default", "default")
+	// client.SetBasicAuth("default", "default")
+	// apiUser := viper.GetString("beetle.api.username")
+	// apiPass := viper.GetString("beetle.api.password")
+	// apiUser := viper.GetString("beetle.tumblebug.api.username")
+	// apiPass := viper.GetString("beetle.tumblebug.api.password")
+	apiUser := os.Getenv("BEETLE_API_USERNAME")
+	apiPass := os.Getenv("BEETLE_API_PASSWORD")
```

Comment:
```suggestion
	apiUser := viper.GetString("beetle.tumblebug.api.username")
	apiPass := viper.GetString("beetle.tumblebug.api.password")
```

Beetle과 Tumblebug의 인증(API username, password)은 의도적으로 구분되어는 상황입니다. 따라서, 현재로서는 위와 같이 수정 후 머지하도록 하겠습니다.